zebrad/tests/acceptance.rs

```diff
@@ -20,7 +20,91 @@
+//! - `FULL_SYNC_TESTNET_TIMEOUT_MINUTES` env variable: The total number of minutes we
+//! will allow this test to run or give up. Value for the Testnet ful  sync tests.
+//! - `/zebrad-cache` directory: For some sync tests, this needs to be created in
+//! the file system, the created directory should have write permissions.
+//!
+//! Please refer to the documentation of each test for more information.
+//!
+//! ## Lightwalletd tests
+//!
+//! The lightwalletd software is an interface service that uses zebrad or zcashd RPC methods to serve wallets or other applications with blockchain content in an efficient manner.
+//! There are several versions of lightwalled in the form of different forks. The original
+//! repo is <https://github.com/zcash/lightwalletd> but Zecwallet Lite uses a custom fork: <https://github.com/adityapk00/lightwalletd>.
+//! The custom fork from adityapk00 is the one Zebra use for this tests:
+//!
+//! Zebra lightwalletd tests are not all marked as ignored but none will not run unless
+//! at least the `ZEBRA_TEST_LIGHTWALLETD` environment variable is present.
+//! Here are some examples of running each test:
+//!
+//! ```console
+//! $ export ZEBRA_TEST_LIGHTWALLETD=true
+//! $ cargo test lightwalletd_integration -- --nocapture
+//!
+//! $ export ZEBRA_TEST_LIGHTWALLETD=true
+//! $ export ZEBRA_CACHED_STATE_DIR="/path/to/zebra/chain"
+//! $ export LIGHTWALLETD_DATA_DIR="/path/to/lightwalletd/database"
+//! $ cargo test lightwalletd_update_sync -- --nocapture
+//!
+//! $ export ZEBRA_TEST_LIGHTWALLETD=true
+//! $ export ZEBRA_CACHED_STATE_DIR="/path/to/zebra/chain"
+//! $ cargo test lightwalletd_full_sync -- --ignored --nocapture
+//!
+//! $ export ZEBRA_TEST_LIGHTWALLETD=true
+//! $ cargo test lightwalletd_test_suite -- --ignored --nocapture
+//!
+//! $ export ZEBRA_TEST_LIGHTWALLETD=true
+//! $ export ZEBRA_CACHED_STATE_DIR="/path/to/zebra/chain"
+//! $ cargo test fully_synced_rpc_test -- --ignored --nocapture
+//!
+//! $ export ZEBRA_TEST_LIGHTWALLETD=true
+//! $ export ZEBRA_CACHED_STATE_DIR="/path/to/zebra/chain"
+//! $ export LIGHTWALLETD_DATA_DIR="/path/to/lightwalletd/database"
+//! $ cargo test sending_transactions_using_lightwalletd --features lightwalletd-grpc-tests -- --ignored --nocapture
+//!
+//! $ export ZEBRA_TEST_LIGHTWALLETD=true
+//! $ export ZEBRA_CACHED_STATE_DIR="/path/to/zebra/chain"
+//! $ export LIGHTWALLETD_DATA_DIR="/path/to/lightwalletd/database"
+//! $ cargo test lightwalletd_wallet_grpc_tests --features lightwalletd-grpc-tests -- --ignored --nocapture
+//! ```
+//!
+//! ### Notes
+//!
+//! - `ZEBRA_TEST_LIGHTWALLETD` env variable: Needs to be present to run any of the lightwalletd tests.
+//! - `ZEBRA_CACHED_STATE_DIR` env var: The path to a zebra blockchain database.
+//! - `LIGHTWALLETD_DATA_DIR` env variable. The path to a lightwalletd database.
+//! - `--features lightwalletd-grpc-tests` cargo flag. The flag given to cargo to build the source code of the running test.
```

Comment:
Priority: The list of environmental variables is at the bottom, and developers have to read through a lot of repetitive scripts to get to them. This makes it less useful as a quick reference.

Reply:
I am wondering what is the value of this kind of comments if the final review( https://github.com/ZcashFoundation/zebra/pull/4523#pullrequestreview-988669353) is "do it again but different".

Reply:
I think the suggestion it to simply move the description of the variables closer to the top of the comment. I agree that seems a good idea.

Reply:
Was done at https://github.com/ZcashFoundation/zebra/pull/4523/commits/407e95e2b9d9eb01931b6e9e8e9b06de8ce32ac1